runtime/android/core_internal/src/org/xwalk/core/internal/HitTestResultInternal.java

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2016 Intel Corporation. All rights reserved.
```

Comment:
@wuhengzhi The copyright is same as others files just 2016 is diffirent, Is there any problem?